src/markdown-pages/install-with-kurl/removing-minio.md

```diff
@@ -0,0 +1,67 @@
+---
+path: "/docs/install-with-kurl/removing-object-storage"
+date: "2021-12-17"
+weight: 21
+linktitle: "Removing Object Storage"
+title: "Removing Object Storage Dependencies"
```

Comment:
@DanStough Is it okay that I changed these to the more generic "Removing Object Storage"? It felt like we were talking about removing object storage in general but then named this specifically after MinIO.

Reply:
Yeah, I think that's fair. I picked MinIO because I thought that would be what folks would be immediately searching for, but this suggestion is a better general description.